src/device.js

```diff
@@ -233,6 +247,14 @@ class Device extends EventEmitter {
             if(typeof configbits === "number"){//Senic explained that they use config bits
                 buf[10] += configbits;//These are encoded in the unused bits of the buffer
             }
+            if(typeof options === "object"){
+                if(options.onion_skinning == true){
+                    buf[10] += 0b00010000;
```

Comment:
Could you use the consts here instead of duplicating the values in the source file?